src/Adapters.Tests/Etw.Net451.Tests/EtwTelemetryModuleTests.cs

```diff
@@ -0,0 +1,218 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="EtwTelemetryModuleTests.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.ApplicationInsights.EtwTelemetryCollector.Tests
+{
+    using System;
+    using Microsoft.ApplicationInsights.EtwCollector;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.ApplicationInsights.Tests;
+    using Microsoft.ApplicationInsights.Tracing.Tests;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    [TestClass]
+    public class EtwTelemetryModuleTests : IDisposable
+    {
+        private const int NoEventSourcesConfiguredEventId = 1;
+        private const int FailedToEnableProvidersEventId = 2;
+        private const int ModuleInitializationFailedEventId = 3;
+        private const int RequiresToRunUnderPriviledgedAccountEventId = 4;
+
+        private readonly AdapterHelper adapterHelper = new AdapterHelper();
+
+        public void Dispose()
+        {
+            this.adapterHelper.Dispose();
+        }
+
+        private TelemetryConfiguration GetTestTelemetryConfiguration(bool resetChannel = true)
+        {
+            var configuration = new TelemetryConfiguration();
+            configuration.InstrumentationKey = this.adapterHelper.InstrumentationKey;
+            if (resetChannel)
+            {
+                configuration.TelemetryChannel = this.adapterHelper.Channel.Reset();
+            }
+            else
+            {
+                configuration.TelemetryChannel = this.adapterHelper.Channel;
+            }
+            return configuration;
+        }
+
+        [TestMethod]
+        [TestCategory("EtwTelemetryModule")]
+        public void EtwTelemetryModuleDefaultConstructorExists()
```

Comment:
(minor) I do not think it is necessary to prefix all test method names in this class with `EtwTelemetryModule`